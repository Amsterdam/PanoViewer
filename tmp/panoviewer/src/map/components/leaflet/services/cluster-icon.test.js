import createClusterIcon from './cluster-icon';

describe('createClusterIcon', () => {
  it('should render', () => {
    expect(createClusterIcon({
      getChildCount: jest.fn().mockReturnValue(10)
    })).toMatchSnapshot();
  });
});
