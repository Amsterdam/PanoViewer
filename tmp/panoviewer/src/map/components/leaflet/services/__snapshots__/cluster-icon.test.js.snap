// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`createClusterIcon should render 1`] = `
NewClass {
  "_initHooksCalled": true,
  "options": Object {
    "className": "o-highlight-cluster",
    "html": "<div
              aria-label=\\"Cluster met 10 onderdelen\\"
              class=\\"o-highlight-cluster__text\\">
            10
          </div>",
    "iconAnchor": Object {
      "x": 19,
      "y": 19,
    },
    "iconSize": Object {
      "x": 39,
      "y": 39,
    },
  },
}
`;
